docs/organizations/using-coding-standards.md

```diff
@@ -4,24 +4,28 @@ Create coding standards on your organization to define and apply shared tool and
 
 Coding standards help you ensure that Codacy analyzes multiple repositories with the same tool and code pattern configurations. For example, you can use a coding standard to ensure that a group of repositories follow the same security rules or coding conventions.
 
-Each repository can only follow one coding standard at a time. Applying a new coding standard to a repository <span class="skip-vale">unassigns</span> any previously applied coding standard.
+!!! note
```

Comment:
IMO, there is too much content in this admonition... 🤔 
We use an admonition to highlight a specific detail from the remaining content at a particular point. If we include a long piece of content, we miss the highlighting goal. 

An alternative would be to find a better spot in the flow to fit some of the bullets, as @nicklem suggested [here](https://github.com/codacy/docs/pull/1735/files#r1202081969).
Another one would be to keep the first three considerations, which are more descriptive/conceptual about coding standards, out of the admonition, and keep the admonition only for the last one, which refers to a specific precedence detail.
WDYT, @prcr ?

Reply:
Indeed you're right, it seems that we've been adding lots of small details about how the coding standards work to this page and now the page is full of tips, notes, and important admonitions. :scream: 

I isolated the change of adding the new information in this pull request to make it self-contained, and will now open a different pull request for refactoring the information around the page in a tidier way.